modules/ai/assets/js/editor/pages/form-layout/components/attachments/thumbnail.js

```diff
@@ -2,17 +2,25 @@ import { useEffect, useRef } from 'react';
 import { Box } from '@elementor/ui';
 import PropTypes from 'prop-types';
 
-const THUMBNAIL_SIZE = 64;
+export const THUMBNAIL_SIZE = 64;
 
 export const Thumbnail = ( props ) => {
 	const previewRef = useRef( null );
 
 	useEffect( () => {
 		if ( previewRef.current ) {
 			const previewRoot = previewRef.current.firstElementChild;
+
 			const width = previewRoot?.offsetWidth || THUMBNAIL_SIZE;
 			const height = previewRoot?.offsetHeight || THUMBNAIL_SIZE;
 
+			// Keep the aspect ratio
+			previewRoot.style.width = `${ width }px`;
+			previewRoot.style.height = `${ height }px`;
```

Comment:
not sure that I understand the issue, but maybe `object-fit: cover;`?

Reply:
It's HTML scaled to the thumbnail size, so the `object-fit` will not work